bin/atnodes

```diff
@@ -308,6 +312,8 @@ OPTIONS:
     -v            Be verbose.
     -w            Prompt for password (used for login and sudo,
                   could be privided by SSH_BATCH_PASSWORD).
+    -W            Prompt for password (like -w but conflict, just for sudo.
+                  Never use -W together with -w, because -w will be ignored).
```

Comment:
I mean we should add Perl code to explicitly check the coexistence of `-w` and `-W`. If that is the case we should abort and print out an error message. Ignoring one of them is not a good practice IMHO :) I have to check out the code or the usage text to figure out which one is actually in effect.